src/db.rs

```diff
@@ -147,7 +164,11 @@ impl Db {
         let handle = self.keyspaces.read();
         let mut keyspaces = Vec::with_capacity(handle.keys().count());
         for key in handle.keys() {
-            keyspaces.push(Frame::String(key.clone()))
+            keyspaces.push(Frame::String(format!(
+                "{}, {}",
+                String::from_utf8(key.to_vec()).map_err(|e| { ExecuteCommandError::Utf8Error(e.utf8_error()) })?,
```

Comment:
You don't need to convert `Bytes` to `String`, cloning bytes doesnt' cause any allocations as it is zero copy. Converting string to bytes will cause unnecessary allocations. Our strings are binary safe, but if we convert binary data to utf_8 strings that might cause some issues.

Reply:
Sounds good 👍